decidim-participatory_processes/app/models/decidim/participatory_process_private_user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  # An ParticipatoryProcessPrivateUser associate user with participatory process
```

Comment:
Can you improve the docs? Reading this I don't understand why this class is used. Saying something like: "this class gives a given User access to a given private Process", or something like this, would be good. 

Same for assemblies.